circuit-codegen/src/main/kotlin/com/slack/circuit/codegen/CircuitSymbolProcessorProvider.kt

```diff
@@ -132,44 +163,66 @@ private class CircuitSymbolProcessor(
           } else {
             FactoryType.UI
           }
+        val assistedParams =
+          fd.assistedParameters(symbols, logger, screenKSType, factoryType == FactoryType.PRESENTER)
         codeBlock =
           when (factoryType) {
             FactoryType.PRESENTER ->
               CodeBlock.of(
-                "%M·{·%M()·}",
+                "%M·{·%M(%L)·}",
```

Comment:
I see that `%L` is assistedParams and that comes from `fd` which is a KSFunctionDeclaration` but what is that? And what is `fd` short hand for?

Reply:
`fd` is just shorter to write than `functionDeclaration`. A `KSFunctionDeclaration` is KSP's modeling of a kotlin function. A `%L` template is for literals in kotlinpoet: https://square.github.io/kotlinpoet/#l-for-literals. `assistedParams` is a `CodeBlock` of any assisted parameters we're passing on to the factory/constructor/function like (`screen = screen, navigator = navigator`)